runtime/doc/lsp.txt

```diff
@@ -25,24 +25,37 @@ Nvim provides an LSP client, but the servers are provided by third parties.
 Follow these steps to get LSP features:
 
 1. Install language servers using your package manager or by following the
-   upstream installation instruction. You can find language servers here:
+   upstream installation instructions. You can find language servers here:
    https://microsoft.github.io/language-server-protocol/implementors/servers/
 
 2. Configure the LSP client per language server. See |vim.lsp.start()| or use
```

Comment:
I'm not a fan of this first sentence. If this is meant to be an introductory explanation, we should remain consistent of who's the client and who's the server.

Reply:
Which sentence in particular? Do you have a suggestion on how to reword it?